packages/gatsby/package.json

```diff
@@ -13,22 +13,17 @@
     "@babel/code-frame": "^7.0.0",
     "@babel/core": "^7.0.0",
     "@babel/parser": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
     "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.0.0",
     "@babel/traverse": "^7.0.0",
     "@reach/router": "^1.1.1",
     "autoprefixer": "^8.6.5",
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^8.2.2",
     "babel-loader": "8.0.0-beta.4",
+    "babel-preset-gatsby": "^0.1.0",
```

Comment:
I added this here manually so that yarn workspaces can resolve it. We need to make sure that Gatsby site pull in that dependency - this looks right to me but could use an additional pair of 👀